bundles/org.openhab.binding.pentair/README.md

```diff
@@ -208,6 +208,7 @@ sitemap pool label="Pool stuff" {
 
 Setting up RS485 and basic protocol - https://www.sdyoung.com/home/decoding-the-pentair-easytouch-rs-485-protocol/
 ser2sock GitHub - https://github.com/nutechsoftware/ser2sock
+Alternative Linux daemon to control Aqualink RS pool controllers. Provides web UI, MQTT client & HTTP API endpoints - https://github.com/sfeakes/AqualinkD
```

Comment:
```suggestion
Alternative Linux daemon to control Aqualink RS pool controllers.
Provides web UI, MQTT client & HTTP API endpoints - https://github.com/sfeakes/AqualinkD
```
Each sentence should be on its own line, this makes it cleaner when someone submits a change.